server/src/test/java/org/candlepin/policy/js/compliance/hash/ComplianceStatusHasherTest.java

```diff
@@ -47,7 +47,7 @@
     private String initialHash;
     private Owner owner;
 
-    @Before
+    @BeforeEach
```

Comment:
+1 Please switch the Assert imports as well.

Reply:
Done.